FtpServer/Main.cpp

```diff
@@ -5,18 +5,111 @@
 #include <thread>
 #include <utility>
 #include <boost/asio.hpp>
+#include <boost/filesystem.hpp>
+#include <fstream>
 
 using boost::asio::ip::tcp;
 
-const int max_length = 1024;
+const int max_length = 1030;
+
+struct Executor
+{
+	static void ChangeDir(tcp::socket &sock)
+	{
+		boost::system::error_code error;
+		std::vector<char> data(max_length);
+		size_t length = sock.read_some(boost::asio::buffer(data), error);
+		data[length] = '\0';
+
+		std::cout << "Dispathed change dir" << std::endl;
+		std::cout << &data[0] << std::endl;
+	}
+
+	static void DownloadFile(tcp::socket &sock)
+	{
+		boost::system::error_code error;
+		std::vector<char> data(max_length);
+		size_t length = sock.read_some(boost::asio::buffer(data), error);
+		data[length] = '\0';
+
+		std::cout << "Dispathed download file" << std::endl;
+		std::cout << &data[0] << std::endl;
+	}
+
+	static void UploadFile(tcp::socket &sock)
+	{
+		boost::system::error_code error;
+		std::vector<char> directory(max_length);
+
+		size_t length = sock.read_some(boost::asio::buffer(directory), error);
+
+		std::string path(&directory[4]);
+		boost::filesystem::path filePath(path);
+		if (exists(filePath))
```

Comment:
Same thing here. We can't receive a big file(more than 1024 bytes). Encapsulate operations as it was done in other pull request.